### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by jgsjnr** [Sophia-Community/SophiApp#4] [Ideas]
Can I help with translations on this project, if the project has someone language/strings file or something like...

**Comment by farag2** [comment]
You're welcome! But only when it's done. :)

**Reply by jgsjnr** [reply]
Ok thks

**Reply by farag2** [reply]
According to your faith be it done to you!

**Reply by uDEV2019** [reply]
> But only when it's done. :)

But please give the translators the opportunity to translate before final release.

**Reply by farag2** [reply]
I will. :)

**Comment by farag2** [comment]
Be patient. 👍

**Reply by uDEV2019** [reply]
Is there already a schedule when translators can start?

**Reply by farag2** [reply]
https://t.me/sophia_chat/64953

**Comment by farag2** [comment]
I'll release the first public release now (in a minute). Unfortunately, no. Wait until the full release.

**Comment by marcosgt3** [comment]
Could you add the Spanish language please or can you explain if there is a way to add it
